app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -39,27 +37,45 @@
       data_attributes[:locale] = unique_locales[0]
     end
   end
+
 %>
 <% if items.any? %>
   <%= tag.div(class: accordion_classes, id: id, data: data_attributes) do %>
     <% items.each_with_index do |item, i| %>
       <%
+        # Nunjucks starts a loop on 1 and the client side JavaScript also
+        # adopts this behaviour. To prevent things from breaking, the index
+        # here also need to be increase by one. (Nunjucks is used by GOV.UK
+        # Frontend, which this component is based on.)
         index = i + 1
 
-        section_classes = %w(gem-c-accordion__section)
-        section_classes << 'gem-c-accordion__section--expanded' if item[:expanded]
+        item[:data_attributes] ||= nil
 
-        summary_classes = %w(gem-c-accordion__section-summary govuk-body)
-      %>
+        section_classes = %w(govuk-accordion__section)
+        section_classes << 'govuk-accordion__section--expanded' if item[:expanded]
 
-      <%= tag.section(class: section_classes) do %>
-        <div class="gem-c-accordion__section-header">
-          <%= content_tag(shared_helper.get_heading_level, class: 'gem-c-accordion__section-heading', id: item[:heading][:id]) do %>
-            <%= tag.span(item[:heading][:text], id: "#{id}-heading-#{index}", data: item[:data_attributes], class: 'gem-c-accordion__section-button') %>
-          <% end %>
+        summary_classes = %w(govuk-accordion__section-summary govuk-body)
+      %>
+      <%= tag.div(class: section_classes) do %>
+        <div class="govuk-accordion__section-header">
```

Comment:
Just wondering if it's worth swapping the `div` markup for `<%= tag.div(class: govuk-accordion__section-header) do %>` to keep consistent with the use of `tag` helpers elsewhere in the view.

Reply:
> This seems okay to me - the only thing I could find visually is that the chevrons are missing in IE8 (screenshots attached below) - but otherwise looks fine and works as expected.

Thanks for this, something most not be porting quite right as IE8 fallback should display an icon, let me review.

Reply:
Personally I wouldn't worry about the icons on IE8 - I think the meaning of the control is conveyed enough with the text.

Reply:
+1 none issue

After some investigation it appears it's (unexpectedly) coming from the other end. 
https://github.com/alphagov/govuk-frontend/issues/2530